rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pipeline-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: pipeline_alerts
+      interval: 1m
+      rules:
+        - alert: HighSchedulingOverhead
+          expr: |
+            (sum(sum_over_time(pipelinerun_duration_scheduled_seconds_sum{status='succeded'}[30m]))
+            /
+            sum(sum_over_time(pipelinerun_duration_scheduled_seconds_count{status='succeded'}[30m])))
+            /
+            (sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_sum{status='success'}[30m]))
+            /
+            sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_count{status='success'}[30m])))  > 0.05
+          for: 12h
```

Comment:
fyi this duration is intentiional ... we expect bumps in overhead from time to time, but want to see prolonged durations of higher overload before we start increasing tekton's minimal CPU/mem requirements or adding replicas of the controller/webhook